src/ui/css/yakyak/colors.less

```diff
@@ -36,6 +38,7 @@ html{
       --lightblack: #475866;
       --black: #A2A2A2;
       --green: #161c20;
+      --online: var(--yakyakgreen);
```

Comment:
exception...

note: `--green` itself could not be redefined as it sets the color of the top bar (that would be a major break)